net-misc/frr/frr-8.0.ebuild

```diff
@@ -23,6 +23,7 @@ COMMON_DEPEND="
 	${PYTHON_DEPS}
 	acct-user/frr
 	dev-libs/json-c:0=
+	net-dns/c-ares:=
```

Comment:
I don't think we need to pull in c-ares unconditionally.
See https://github.com/FRRouting/frr/pull/9415

Reply:
It’s 5 days old, the 8.0 is a month old

Reply:
Yeah but I would like to backport that patch to 8.0

Reply:
I added the patch and the optfeature in my last commit